gax/build.gradle

```diff
@@ -5,7 +5,8 @@ project.version = "2.12.3-SNAPSHOT" // {x-version-update:gax:current}
 
 dependencies {
   api(libraries['maven.com_google_api_api_common'],
-    libraries['maven.com_google_auth_google_auth_library_credentials'],
+          libraries['maven.com_google_api_grpc_proto_google_common_protos'],
+          libraries['maven.com_google_auth_google_auth_library_credentials'],
```

Comment:
Why is this one needed?

Reply:
`com_google_auth_google_auth_library_credentials`? I think It's a formatting/git issue, I added `com_google_api_grpc_proto_google_common_protos` but somehow git thinks I removed one and added two.